src/attr/__init__.pyi

```diff
@@ -49,19 +53,26 @@ def fields(cls: type) -> Tuple[Attribute, ...]: ...
 def validate(inst: Any) -> None: ...
 
 # we use Any instead of _CountingAttr so that e.g. `make_class('Foo', [attr.ib()])` is valid
-def make_class(name, attrs: Union[List[Any], Dict[str, Any]], bases: Tuple[type, ...] = ..., **attributes_arguments) -> type: ...
+def make_class(name, attrs: Union[List[str], Dict[str, Any]], bases: Tuple[type, ...] = ..., **attributes_arguments) -> type: ...
```

Comment:
If we wanted to be more explicit you could replace `**attributes_arguments` with all the arguments fro the attributes call.  It does mean that if you add another argument you have to add it here. But then you'll get better type checking.

Reply:
True.  I'll make that change.